### PR TITLE
[merge into branch theme-updates] add/remove tokens in theme; update Pill component

### PR DESCRIPTION
**Pull request by feliciackh**
Theme
- add new tokens to characteristics/text.css file and characteristics/text.stories.mdx (excel line 65, 66)
- add new token to foundations/size.css (excel line 76)
- remove all characteristics/taggable.css tokens (excel line 58) #2014

Component
- update Pill component to use tokens from design spec (excel line 58) #2014

### CODE REVIEW

File: CONTRIBUTING.md
Comment (feliciackh) [inline CONTRIBUTING.md:1]: prettier only

File: .changeset/tall-buckets-change.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+---
+
+- New tokens added for `text` characteristics
+
+```diff
++ --salt-text-notation-fontWeight-small: var(--salt-typography-fontWeight-regular);
++ --salt-text-notation-fontWeight-strong: var(--salt-typography-fontWeight-bold);
+```
+
+- New token added for `size` foundations
```
Comment (libertymayc) [inline .changeset/tall-buckets-change.md:12]:
```suggestion
- New token added to `size` foundations
```

File: .changeset/tall-buckets-change.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+---
+"@salt-ds/theme": minor
+---
+
+- New tokens added for `text` characteristics
```
Comment (libertymayc) [inline .changeset/tall-buckets-change.md:5]:
```suggestion
- New tokens added to `text` characteristics
```

File: packages/theme/css/foundations/size.css [hunk 3]
```diff
@@ -7,7 +7,6 @@
   --salt-size-indicator: 1px;
   --salt-size-selectable: 12px;
   --salt-size-bar-strong: 4px;
-  /* --salt-size-bar-small: 4px; */
```
Comment (feliciackh) [inline packages/theme/css/foundations/size.css:10]: this token was added/commented out in the `theme-updates` branch, removing it in this PR as no longer applicable

File: packages/theme/css/foundations/size.css [hunk 4]
```diff
@@ -19,7 +18,6 @@
   --salt-size-indicator: 2px;
   --salt-size-selectable: 14px;
   --salt-size-bar-strong: 8px;
-  /* --salt-size-bar-small: 4px; */
```
Comment (feliciackh) [inline packages/theme/css/foundations/size.css:22]: this token was added/commented out in the `theme-updates` branch, removing it in this PR as no longer applicable

File: packages/theme/css/foundations/size.css [hunk 5]
```diff
@@ -31,7 +29,6 @@
   --salt-size-indicator: 3px;
   --salt-size-selectable: 16px;
   --salt-size-bar-strong: 12px;
-  /* --salt-size-bar-small: 4px; */
```
Comment (feliciackh) [inline packages/theme/css/foundations/size.css:34]: this token was added/commented out in the `theme-updates` branch, removing it in this PR as no longer applicable

File: packages/theme/css/foundations/size.css [hunk 6]
```diff
@@ -43,12 +40,12 @@
   --salt-size-indicator: 4px;
   --salt-size-selectable: 18px;
   --salt-size-bar-strong: 16px;
-  /* --salt-size-bar-small: 4px; */
```
Comment (feliciackh) [inline packages/theme/css/foundations/size.css:46]: this token was added/commented out in the `theme-updates` branch, removing it in this PR as no longer applicable

File: .changeset/tall-buckets-change.md [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
++ --salt-text-notation-fontWeight-strong: var(--salt-typography-fontWeight-bold);
+```
+
+- New token added for `size` foundations
+
+```diff
++ --salt-size-bar-small: 2px;
+```
+
+- Deprecated all tokens in `taggable` characteristics
```
[the remark this answers is not in the document]
Comment (feliciackh) [reply .changeset/tall-buckets-change.md:18]: updated to remove values of the tokens. Also for `taggable` specifically there isn't any replacement tokens, what's being used in Pill is a separate update where design specs have been updated. I've added in changeset no replacement tokens for taggable

File: .changeset/tall-buckets-change.md [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+-  --salt-taggable-background-active: var(--salt-palette-interact-primary-background-active);
+-  --salt-taggable-background-disabled: var(--salt-palette-interact-primary-background-disabled);
+
+-  --salt-taggable-foreground: var(--salt-palette-interact-primary-foreground);
+-  --salt-taggable-foreground-hover: var(--salt-palette-interact-primary-foreground-hover);
+-  --salt-taggable-foreground-active: var(--salt-palette-interact-primary-foreground-active);
+-  --salt-taggable-foreground-disabled: var(--salt-palette-interact-primary-foreground-disabled);
+```
+
+- Updated `taggable` characteristics tokens used in `@salt-ds/lab` Pill component to use `actionable` characteristics tokens
```
[the remark this answers is not in the document]
Comment (feliciackh) [reply .changeset/tall-buckets-change.md:36]: as above, taggable has no direct replacement tokens. I've moved mentions of Pill component changes into the lab component changeset

File: .changeset/tall-buckets-change.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (libertymayc) [reply .changeset/tall-buckets-change.md]: replacement would then be the calculated value that the palette trickles down to

**Review by lilyvc — APPROVED** [review]

File: .changeset/tall-buckets-change.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (feliciackh) [reply .changeset/tall-buckets-change.md]: @libertymayc please have a look!

**Pull request merged**